packages/@aws-cdk/aws-appmesh/lib/gateway-route-spec.ts

```diff
@@ -7,36 +12,153 @@ import { IVirtualService } from './virtual-service';
 import { Construct } from '@aws-cdk/core';
 
 /**
- * The criterion for determining a request match for this GatewayRoute
+ * Configuration for gateway route host name match.
+ */
+export interface GatewayRouteHostnameMatchConfig {
+  /**
+   * GatewayRoute CFN configuration for host name match.
+   */
+  readonly hostnameMatch: CfnGatewayRoute.GatewayRouteHostnameMatchProperty;
+}
+
+/**
+ * Used to generate host name matching methods.
+ */
+export abstract class GatewayRouteHostnameMatch {
+  /**
+   * The value of the host name must match the specified value exactly.
+   *
+   * @param name The exact host name to match on
+   */
+  public static exact(name: string): GatewayRouteHostnameMatch {
```

Comment:
You're probably right that `exactly` is a better name here - it's closer to `endingWith`. "Exact" just sounds a little bit off.

Reply:
Changed to `exactly`